zio-http/src/test/scala/zhttp/service/ServerSpec.scala

```diff
@@ -262,4 +269,32 @@ object ServerSpec extends HttpRunnableSpec {
         } yield assertTrue(data == Status.OK)
       }
   }
+
+  def validationAppSpec = suite("ValidationAppSpec") {
+    testM("200 response GET") {
+      val actual = status(!! / "HExitSuccess")
+      assertM(actual)(equalTo(Status.OK))
+    } +
+      testM("200 response POST") {
+        val actual = status(!! / "HExitSuccess", Method.POST)
+        assertM(actual)(equalTo(Status.OK))
+      } +
```

Comment:
combine these two test using `HttpGen.method` and `checkAllM`

Reply:
done